android/app/src/main/assets/map.json

```diff
@@ -0,0 +1,143 @@
+{
+"map":
+[
+  {
```

Comment:
Why are you defining a new json. We already have one by the name microlocations in the assets.